dvc/repo/ctx.py

```diff
@@ -0,0 +1,14 @@
+def ctx(method):
```

Comment:
There should be a better name ). Also, you are loosing method name, doc-strings, etc - use `@functools.wraps()`.